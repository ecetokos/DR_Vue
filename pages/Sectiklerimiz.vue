<template>
    <div class="kapsayici">      
  <section class="sub-categories">
    <div class="categories-path">
        <div class="container">
            <ul style="padding:30px;">
                <li>
                    <div> 
                        <a href="/">ANASAYFA    </a>
                        <img style="width:10px" src="https://www.flaticon.com/svg/static/icons/svg/758/758778.svg"/>                     
                    </div>
                </li>
                 <li><h1><span style="visibility:hidden;">xx</span>SİZİN İ&#199;İN SE&#199;TİĞİMİZ KİTAPLAR!..</h1></li>
            </ul>
        </div>
    </div>
    </section>
<section class="filter">
    <div class="sc-filter">
        <div class="container" >       
            <div class="row" >
                <div class="filter"> <!-- .filter-media-->
                <div class="fltr" >
                    <select name="" id="media" class="flt custom select-box-filter"  >
                        <option  value=""  >MEDYA CİNSİ  <pre>   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &#11206; </pre>   </option>
                            <option id="media_1134" value="İnce Kapak">İnce Kapak </option>
                            <option id="media_609" value="Ciltli">Ciltli   </option>
                            <option id="media_610" value="Cep Boy">Cep Boy </option>
                            <option id="media_789" value="3 Aylık">3 Aylık </option>                           
                   </select>
                   <!--
                       <pre>   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &#9205; </pre>
                     <a href=""> <img src="https://cdn4.iconfinder.com/data/icons/mayssam/512/right-512.png" height="10" width="10">   </a>
                     -->
                </div>                 
                </div>
                <div class="filter" id="priceFilter"> <!--.filter-price-->
                <div class="fltr">
                    <select name="" id="price" class="flt custom select-box-filter priceControl">
        <option value="">FİYAT  <pre>   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &#11206; </pre> </option>
            <option value="0,25">0 TL - 25 TL</option>
            <option value="25,50">25 TL - 50 TL</option>
            <option value="50,100">50 TL - 100 TL</option>
            <option value="100,250">100 TL - 250 TL</option>
            <option value="250,500">250 TL - 500 TL</option>
            <option value="500,-1">500 TL ve &#252;zeri</option>
    </select>
                </div>
    
</div>           
                <div class="filter-order-by" >
                    <div class="fltr" >
                    <select name="" id="order-by" class="custom select-box-filter flt" autocomplete="off" >             
                            <option value="groups.group.displayorder,asc" selected="selected" >Seçiniz  <pre>   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &#11206; </pre> </option>
                            <option value="price,desc">Azalan Fiyat</option>
                            <option value="price,asc">Artan Fiyat</option>
                            <option value="soldcount,desc">Çok Satan</option>
                            <option value="newness,desc">Yeni</option>
                            <option value="newness,asc">Eski</option>
                            <option value="name.sortable,asc">A - Z</option>
                            <option value="name.sortable,desc">Z - A</option>
                            <option value="discount,desc">Azalan İndirim</option>
                            <option value="discount,asc">Artan İndirim</option>             
                    </select>
                    <!--
                    <img style="width:5px" src="https://www.flaticon.com/svg/static/icons/svg/758/758778.svg"/>
                    -->
                    </div>                   
                </div>
            <div class="row">
                        <div style="visibility:hidden;">
                            <div>
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                            </div>
                         </div>
                <div class="filter-results-count"  style="float:right;">
                    
                         <span id="searchResultCount">5038 ADET</span>
                </div>
            </div>                
<div class="container">
    <ul class="pager"><li><a class="active" href="javascript:;">1</a></li><li href="#"><a href="javascript:;">2</a></li><li href="#"><a href="javascript:;">3</a></li><li href="#"><a href="javascript:;">4</a></li><li><a href="javascript:;">...</a></li><li href="#"><a href="javascript:;">119</a></li><li href="#"><a href="javascript:;"></a><a class="next" href="javascript:;"> </a>
    </li>   <a href=""> <img src="https://cdn4.iconfinder.com/data/icons/mayssam/512/right-512.png" height="10" width="10">   </a> </ul>

</div>
            </div>
        </div>
    </div>
</section>
    </div>
</template>


<script>
  export default {
    name: 'RankingBar'
  }
  
</script>

<style scoped lang="scss">
@import "../src/css/dynamic-banner.css";
@import "../src/css/dr-custom.css";
@import "../src/css/style.css";
.fltr{
    width: 150px;
    border: 1px lightgrey solid;
    padding: 10px;
}
select{
    overflow: auto;
}
</style>