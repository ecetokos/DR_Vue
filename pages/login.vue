<template>
    <div>
        <Login/>   
        <Hesabim/>     
    </div> 
</template>

<script>
import Login from '../src/components/Login';
import Hesabim from '../src/components/Hesabim'

export default {
  components: {
    Login,
    Hesabim
  },
};
</script>