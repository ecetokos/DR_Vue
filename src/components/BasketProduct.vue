<template>
<div class="header-new">
    <div class="head-menu"><div>{{basketitems}}</div>
        <div class="head-cart active">  
            <div v-if="denemequan==0" style="text-align: center;">
                Sepetinizde ürün bulunmamaktadır.
            </div>        
       <div v-else>    
     <ul id="products">
         <li v-for="(product,index) in basketproducts" v-bind:key="product" >
             <figure class="foto">
                 <a href="/Kitap/Son-Curet/Yilmaz-Ozdil/Arastirma-Tarih/Politika-Arastirma/Turkiye-Politika-/urunno=0001889645001">
                    <img :src="product.img" style="height:85px" alt="Son Cüret"></a>
             </figure>
             <div class="detail">
                 <p class="product-name"><a title="Son Cüret" href="/Kitap/Son-Curet/Yilmaz-Ozdil/Arastirma-Tarih/Politika-Arastirma/Turkiye-Politika-/urunno=0001889645001">{{product.name}}</a></p>
                 <p class="product-type"><a title="Son Cüret" href="/Kitap/Son-Curet/Yilmaz-Ozdil/Arastirma-Tarih/Politika-Arastirma/Turkiye-Politika-/urunno=0001889645001">Medya Tipi: {{product.kapak}} / {{product.quantity}} ADET</a></p>
                 <p class="product-price"><a title="Son Cüret" href="/Kitap/Son-Curet/Yilmaz-Ozdil/Arastirma-Tarih/Politika-Arastirma/Turkiye-Politika-/urunno=0001889645001" style="color:black">{{(product.price*product.quantity).toFixed(2)}} TL</a></p>
             </div>
             <span style="display:none" id="index">{{index}}</span>
         </li>
     </ul>
     <div class="sum">
         <p class="sum-detail" id="totalquantity">TOPLAM {{totalquantityy}} ÜRÜN</p>
         <span class="sum-price" id="totalprice">{{totalprice.toFixed(2)}} TL</span>                
     </div>
     </div> 
        <nuxt-link to="/Sepetim"><input class="btn red buton" value="SEPETE GİT" type="button"></nuxt-link>
</div>
    </div>
    
    
</div>
    
</template>

<script>
import store from '../../store/index'
  export default {
    name: 'BasketProduct',
    data(){
        return{
            basketproducts:[],
            basketproductss:[],
            totalquan:0,
            denemequan:0,
            totalquantityy:0,
            totalprice:0
        }
    },
    computed:{
        /*basketproducts(){
            return this.$store.getters.cartProducts
        },*/
        /*total(){
            return this.$store.getters.cartTotal
        },*/
        quantitytotal(){
            var quantotal=0
            //return this.$store.getters.totalQuantity
            if(this.authUser && this.basketproducts){                      
                //alert(this.basketitemss[0].quantity)
                //alert(this.basketproducts[0].quantity)
                //alert(this.basketproducts[i].quantity)
                for(var i=0;i<this.denemequan;i++){
                    this.totalquantity += this.basketproducts[i].quantity
                }
                /*this.basketproducts.forEach(function (product){
                    quantotal+=product.quantity
                })*/
                //return this.basketproducts.reduce((total,product)=>total+product.quantity,0)
                
                return quantotal
            }
            else{return quantotal}
        },
        authUser(){     
          return this.$store.getters.takeuser
       },
       basketitems(){ 
           var sayi=0
           var toplam=0
           if(this.authUser){
            this.$fire.database.ref('/users/'+this.authUser.uid+'/basketitems').on('value',(snapshot)=>{      
                this.basketproductss=snapshot.val()
                this.denemequan=Object.keys(this.basketproductss).length
                if(this.denemequan>0){
                    snapshot.forEach((element) => {
                        sayi+=element.val().quantity 
                        toplam+=element.val().quantity*element.val().price       
                    })
                    this.totalquantityy=sayi
                    this.totalprice=toplam
                } 
            })
            if(this.basketproductss){
              this.basketproducts=this.basketproductss
            }
           }  
       }
    }
  }
</script>

<style scoped lang="scss">
@import "../css/dynamic-banner.css";
@import "../css/dr-custom.css";
@import "../css/style.css";
.header-new{
    background-color: white;
}
.head-cart{
    border: 1px solid;
    border-radius: 2px;
}
.foto{
    float: left;
    margin-left: 10px;
    margin-top: 11px;
    margin-right: 10px;
}
li{
    height: 110px;
    border: solid 1px lightgrey;
    
}
.detail{
    margin-top: 15px;
}
.sum{
    text-align: center;
    align-content: center;
}
.head-cart{
    width: 320px;
    padding: 20px;
}
.buton{
    width: 275px;
    align-content: center;
    margin-top: 10px;
}

.sum-detail
{
    font-size: 10px;
    color: gray;
    margin-top: 5px;
}
.sum-price{
    font-size: 20px;
}
.product-type{
    font-size: 11px;
    font-weight: bold;
}
hr{
    margin-left: 120px;
}
</style>