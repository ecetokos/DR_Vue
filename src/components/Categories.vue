<template>
    <div class="categoriess">
                <ul >
                    <li  style="margin-left:120px;width:8%;" ><a href="/kategori/Kitap"><img style="height:70px " src="https://www.flaticon.com/svg/static/icons/svg/864/864685.svg" class="icon-kitap" /></a><br/> <a href="/kategori/Kitap"><span>Kitap <span style="visibility:hidden;">xxxxx</span> </span></a></li>
                    <span  class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:8%" ><a href="/kategori/EKitap"><img style="height:70px" src="https://www.dr.com.tr/Themes/DR/Content/assets/images/general/kobo-menu-logo.png" class="icon-kitap" /></a><br/> <a href="/kategori/EKitap"><span>E-Kitap <span style="visibility:hidden;">xxxx</span></span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:8%"><a href="/kategori/Elektronik"><img style="height:70px" src="https://www.flaticon.com/svg/static/icons/svg/685/685655.svg" class="icon-kitap" /></a><br/> <a href="/kategori/Elektronik"><span>Elektronik <span style="visibility:hidden;">xxxx</span></span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="margin-top:50px;width:8%;" ><a href="/kategori/Hobi-Oyuncak"><img style="height:80px;" src="https://www.flaticon.com/svg/static/icons/svg/334/334998.svg" class="icon-kitap" /></a><br/> <a href="/kategori/Hobi-Oyuncak"><span>Hobi & <br/>Oyuncak</span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:8%"><a href="/kategori/Kirtasiye"><img style="height:70px" src="https://www.flaticon.com/svg/static/icons/svg/1250/1250615.svg" class="icon-kitap" /></a><br/> <a href="/kategori/Kirtasiye"><span>Kırtasiye <span style="visibility:hidden;">xxxx</span></span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:8%"><a href="/kategori/Oyun-Konsol"><img style="height:70px;"  src="https://www.flaticon.com/svg/static/icons/svg/686/686589.svg" class="icon-kitap" /></a><br/> <a href="/kategori/Oyun-Konsol"><span>Oyun & Konsol</span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:8%"><a href="/kategori/Muzik"><img style="height:70px;" src="https://www.flaticon.com/svg/static/icons/svg/126/126493.svg" class="icon-kitap" /></a><br/> <a href="/kategori/Muzik"><span>Müzik <span style="visibility:hidden;">xxxx</span></span></a></li>
                   <!-- <hr class="solid"> -->
                    <hr class="vl2">
                    <li style="margin-left:110px;width:8%;" ><a href="/kataloglar_/turkuvaz-dergiler/7007#/page=1/sort=soldcount,desc/categoryid=0/clog=7007/parentId=0/price=-1,-1"><img style="height:85px" src="https://www.dr.com.tr/Themes/DR/Content/assets/images/general/dergi-icon.jpg" class="icon-kitap" /></a><br/> <a href="/kataloglar_/turkuvaz-dergiler/7007#/page=1/sort=soldcount,desc/categoryid=0/clog=7007/parentId=0/price=-1,-1"><span>Dergi <span style="visibility:hidden;">xxxxx</span></span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:8%"><a href="/kategori/Kisisel-Urunler"><img  style="height:70px;" src="https://www.flaticon.com/svg/static/icons/svg/620/620875.svg" class="icon-kitap" /></a><br/> <a href="/kategori/Kisisel-Urunler"><span>Hediyelik<br/>Eşyalar </span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:8%"><a href="/kategori/Spor-Outdoor"><img style="height:70px;" src="https://www.flaticon.com/svg/static/icons/svg/76/76696.svg" class="icon-kitap" /></a><br/> <a href="/kategori/Spor-Outdoor"><span>Spor &<br/> Outdoor</span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:8%"><a href="/LP-top20"><img style="height:70px;" src="https://www.flaticon.com/svg/static/icons/svg/182/182596.svg" class="icon-kitap" /></a><br/> <a href="/LP-top20"><span>Çok Satanlar</span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:9%"><a href="/Kategori_/Kitap/En-Yeniler/10001/3"><img style="height:70px;" src="https://www.flaticon.com/svg/static/icons/svg/3618/3618186.svg" class="icon-kitap" /></a><br/> <a href="/Kategori_/Kitap/En-Yeniler/10001/3"><span>Yeni Çıkan<br/> Kitaplar</span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:8%"><a href="/yazarlar"><img style="height:80px;" src="https://www.dr.com.tr/Themes/DR/Content/assets/images/general/yazarlar-icon.jpg" class="icon-kitap" /></a><br/> <a href="/yazarlar"><span>Yazarlar <span style="visibility:hidden;">xxxx</span></span></a></li>
                    <span class="vl" style="font-size: 1000%; background: inherit;"></span>
                    <li style="width:8%"><a href="/yayinevleri"><img style="height:80px;" src="https://www.dr.com.tr/Themes/DR/Content/assets/images/general/yayinevleri-icon.jpg" class="icon-kitap" /></a><br/> <a href="/yayinevleri"><span>Yayınevleri <span style="visibility:hidden;">xxxx</span></span></a></li>
                </ul>
            </div>
</template>

<script>
  export default {
    name: 'Categories'
  }
</script>

<style scoped lang="scss">
@import "../css/dynamic-banner.css";
@import "../css/dr-custom.css";
@import "../css/style.css";
.categoriess{
  background-color: white;
}
            .categoriess li{   
                display: inline-block;
                margin: 33px;
                text-align: center;
            }
            .categoriess li span{
                font-size: 25px;
                color: black;
            }
            .vl {
             border-left: 1.6px solid #e6e6e6;
             height: 300px;
}
.vl2 {
              border: 1px solid #e6e6e6;
              margin-top: 0px;
              margin-bottom: 0px ;
}

</style>