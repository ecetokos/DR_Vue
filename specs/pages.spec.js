import { mount} from '@vue/test-utils'
import Sectiklerimiz from '../pages/Sectiklerimiz.vue'
import rankingbar from '../src/components/RankingBar.vue'
import Login from '../pages/login.vue'
import login from '../src/components/Login.vue'
import Navbar from '../src/components/Navbar.vue'
import Categories from '../src/components/Categories.vue'
import BasketProduct from '../src/components/BasketProduct.vue'
import Store from '../store/index'
import Vue from 'vue'
import Vuex from 'vuex'
Vue.use(Vuex)

test('does login page contain components',()=>{
    const store=new Vuex.Store(Store)
    const wrappernb=mount(Login,{store})
    expect(wrappernb.contains(login)).toBe(true)
})

test('does Sectiklerimiz page contain components',()=>{
    const store=new Vuex.Store(Store)
    const wrappernb=mount(Sectiklerimiz,{store})
    expect(wrappernb.contains(rankingbar)).toBe(true)
})

test('does navbar contain components',()=>{
    const store=new Vuex.Store(Store)
    const wrappernb=mount(Navbar,{store})
    expect(wrappernb.contains(Categories)).toBe(true)
    expect(wrappernb.contains(BasketProduct)).toBe(true)
})

test('has the expected html structure',()=>{
    const wrapperbp=mount(rankingbar)
    expect(wrapperbp.element).toMatchSnapshot()
    
    const wrapperc=mount(Categories)
    expect(wrapperc.element).toMatchSnapshot()
})
