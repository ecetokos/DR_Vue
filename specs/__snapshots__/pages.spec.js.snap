// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has the expected html structure 1`] = `
<div
  class="kapsayici"
>
  <section
    class="sub-categories"
  >
    <div
      class="categories-path"
    >
      <div
        class="container"
      >
        <ul
          style="padding: 30px;"
        >
          <li>
            <div>
              <a
                href="/"
              >
                ANASAYFA    
              </a>
               
              <img
                src="https://www.flaticon.com/svg/static/icons/svg/758/758778.svg"
                style="width: 10px;"
              />
            </div>
          </li>
           
          <li>
            <h1>
              <span
                style="visibility: hidden;"
              >
                xx
              </span>
              SİZİN İÇİN SEÇTİĞİMİZ KİTAPLAR!..
            </h1>
          </li>
        </ul>
      </div>
    </div>
  </section>
   
  <section
    class="filter"
  >
    <div
      class="sc-filter"
    >
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="filter"
          >
            <div
              class="fltr"
            >
              <select
                class="flt custom select-box-filter"
                id="media"
                name=""
              >
                <option
                  value=""
                >
                  MEDYA CİNSİ  
                  <pre>
                             ⯆ 
                  </pre>
                </option>
                 
                <option
                  id="media_1134"
                  value="İnce Kapak"
                >
                  İnce Kapak 
                </option>
                 
                <option
                  id="media_609"
                  value="Ciltli"
                >
                  Ciltli   
                </option>
                 
                <option
                  id="media_610"
                  value="Cep Boy"
                >
                  Cep Boy 
                </option>
                 
                <option
                  id="media_789"
                  value="3 Aylık"
                >
                  3 Aylık 
                </option>
              </select>
            </div>
          </div>
           
          <div
            class="filter"
            id="priceFilter"
          >
            <div
              class="fltr"
            >
              <select
                class="flt custom select-box-filter priceControl"
                id="price"
                name=""
              >
                <option
                  value=""
                >
                  FİYAT  
                  <pre>
                                            ⯆ 
                  </pre>
                </option>
                 
                <option
                  value="0,25"
                >
                  0 TL - 25 TL
                </option>
                 
                <option
                  value="25,50"
                >
                  25 TL - 50 TL
                </option>
                 
                <option
                  value="50,100"
                >
                  50 TL - 100 TL
                </option>
                 
                <option
                  value="100,250"
                >
                  100 TL - 250 TL
                </option>
                 
                <option
                  value="250,500"
                >
                  250 TL - 500 TL
                </option>
                 
                <option
                  value="500,-1"
                >
                  500 TL ve üzeri
                </option>
              </select>
            </div>
          </div>
           
          <div
            class="filter-order-by"
          >
            <div
              class="fltr"
            >
              <select
                autocomplete="off"
                class="custom select-box-filter flt"
                id="order-by"
                name=""
              >
                <option
                  selected="selected"
                  value="groups.group.displayorder,asc"
                >
                  Seçiniz  
                  <pre>
                                         ⯆ 
                  </pre>
                </option>
                 
                <option
                  value="price,desc"
                >
                  Azalan Fiyat
                </option>
                 
                <option
                  value="price,asc"
                >
                  Artan Fiyat
                </option>
                 
                <option
                  value="soldcount,desc"
                >
                  Çok Satan
                </option>
                 
                <option
                  value="newness,desc"
                >
                  Yeni
                </option>
                 
                <option
                  value="newness,asc"
                >
                  Eski
                </option>
                 
                <option
                  value="name.sortable,asc"
                >
                  A - Z
                </option>
                 
                <option
                  value="name.sortable,desc"
                >
                  Z - A
                </option>
                 
                <option
                  value="discount,desc"
                >
                  Azalan İndirim
                </option>
                 
                <option
                  value="discount,asc"
                >
                  Artan İndirim
                </option>
              </select>
            </div>
          </div>
           
          <div
            class="row"
          >
            <div
              style="visibility: hidden;"
            >
              <div>
                
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                            
              </div>
            </div>
             
            <div
              class="filter-results-count"
              style="float: right;"
            >
              <span
                id="searchResultCount"
              >
                5038 ADET
              </span>
            </div>
          </div>
           
          <div
            class="container"
          >
            <ul
              class="pager"
            >
              <li>
                <a
                  class="active"
                  href="javascript:;"
                >
                  1
                </a>
              </li>
              <li
                href="#"
              >
                <a
                  href="javascript:;"
                >
                  2
                </a>
              </li>
              <li
                href="#"
              >
                <a
                  href="javascript:;"
                >
                  3
                </a>
              </li>
              <li
                href="#"
              >
                <a
                  href="javascript:;"
                >
                  4
                </a>
              </li>
              <li>
                <a
                  href="javascript:;"
                >
                  ...
                </a>
              </li>
              <li
                href="#"
              >
                <a
                  href="javascript:;"
                >
                  119
                </a>
              </li>
              <li
                href="#"
              >
                <a
                  href="javascript:;"
                />
                <a
                  class="next"
                  href="javascript:;"
                />
              </li>
               
              <a
                href=""
              >
                <img
                  height="10"
                  src="https://cdn4.iconfinder.com/data/icons/mayssam/512/right-512.png"
                  width="10"
                />
              </a>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`has the expected html structure 2`] = `
<div
  class="categoriess"
>
  <ul>
    <li
      style="margin-left: 120px; width: 8%;"
    >
      <a
        href="/kategori/Kitap"
      >
        <img
          class="icon-kitap"
          src="https://www.flaticon.com/svg/static/icons/svg/864/864685.svg"
          style="height: 70px;"
        />
      </a>
      <br />
       
      <a
        href="/kategori/Kitap"
      >
        <span>
          Kitap 
          <span
            style="visibility: hidden;"
          >
            xxxxx
          </span>
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 8%;"
    >
      <a
        href="/kategori/EKitap"
      >
        <img
          class="icon-kitap"
          src="https://www.dr.com.tr/Themes/DR/Content/assets/images/general/kobo-menu-logo.png"
          style="height: 70px;"
        />
      </a>
      <br />
       
      <a
        href="/kategori/EKitap"
      >
        <span>
          E-Kitap 
          <span
            style="visibility: hidden;"
          >
            xxxx
          </span>
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 8%;"
    >
      <a
        href="/kategori/Elektronik"
      >
        <img
          class="icon-kitap"
          src="https://www.flaticon.com/svg/static/icons/svg/685/685655.svg"
          style="height: 70px;"
        />
      </a>
      <br />
       
      <a
        href="/kategori/Elektronik"
      >
        <span>
          Elektronik 
          <span
            style="visibility: hidden;"
          >
            xxxx
          </span>
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="margin-top: 50px; width: 8%;"
    >
      <a
        href="/kategori/Hobi-Oyuncak"
      >
        <img
          class="icon-kitap"
          src="https://www.flaticon.com/svg/static/icons/svg/334/334998.svg"
          style="height: 80px;"
        />
      </a>
      <br />
       
      <a
        href="/kategori/Hobi-Oyuncak"
      >
        <span>
          Hobi & 
          <br />
          Oyuncak
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 8%;"
    >
      <a
        href="/kategori/Kirtasiye"
      >
        <img
          class="icon-kitap"
          src="https://www.flaticon.com/svg/static/icons/svg/1250/1250615.svg"
          style="height: 70px;"
        />
      </a>
      <br />
       
      <a
        href="/kategori/Kirtasiye"
      >
        <span>
          Kırtasiye 
          <span
            style="visibility: hidden;"
          >
            xxxx
          </span>
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 8%;"
    >
      <a
        href="/kategori/Oyun-Konsol"
      >
        <img
          class="icon-kitap"
          src="https://www.flaticon.com/svg/static/icons/svg/686/686589.svg"
          style="height: 70px;"
        />
      </a>
      <br />
       
      <a
        href="/kategori/Oyun-Konsol"
      >
        <span>
          Oyun & Konsol
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 8%;"
    >
      <a
        href="/kategori/Muzik"
      >
        <img
          class="icon-kitap"
          src="https://www.flaticon.com/svg/static/icons/svg/126/126493.svg"
          style="height: 70px;"
        />
      </a>
      <br />
       
      <a
        href="/kategori/Muzik"
      >
        <span>
          Müzik 
          <span
            style="visibility: hidden;"
          >
            xxxx
          </span>
        </span>
      </a>
    </li>
     
    <hr
      class="vl2"
    />
     
    <li
      style="margin-left: 110px; width: 8%;"
    >
      <a
        href="/kataloglar_/turkuvaz-dergiler/7007#/page=1/sort=soldcount,desc/categoryid=0/clog=7007/parentId=0/price=-1,-1"
      >
        <img
          class="icon-kitap"
          src="https://www.dr.com.tr/Themes/DR/Content/assets/images/general/dergi-icon.jpg"
          style="height: 85px;"
        />
      </a>
      <br />
       
      <a
        href="/kataloglar_/turkuvaz-dergiler/7007#/page=1/sort=soldcount,desc/categoryid=0/clog=7007/parentId=0/price=-1,-1"
      >
        <span>
          Dergi 
          <span
            style="visibility: hidden;"
          >
            xxxxx
          </span>
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 8%;"
    >
      <a
        href="/kategori/Kisisel-Urunler"
      >
        <img
          class="icon-kitap"
          src="https://www.flaticon.com/svg/static/icons/svg/620/620875.svg"
          style="height: 70px;"
        />
      </a>
      <br />
       
      <a
        href="/kategori/Kisisel-Urunler"
      >
        <span>
          Hediyelik
          <br />
          Eşyalar 
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 8%;"
    >
      <a
        href="/kategori/Spor-Outdoor"
      >
        <img
          class="icon-kitap"
          src="https://www.flaticon.com/svg/static/icons/svg/76/76696.svg"
          style="height: 70px;"
        />
      </a>
      <br />
       
      <a
        href="/kategori/Spor-Outdoor"
      >
        <span>
          Spor &
          <br />
           Outdoor
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 8%;"
    >
      <a
        href="/LP-top20"
      >
        <img
          class="icon-kitap"
          src="https://www.flaticon.com/svg/static/icons/svg/182/182596.svg"
          style="height: 70px;"
        />
      </a>
      <br />
       
      <a
        href="/LP-top20"
      >
        <span>
          Çok Satanlar
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 9%;"
    >
      <a
        href="/Kategori_/Kitap/En-Yeniler/10001/3"
      >
        <img
          class="icon-kitap"
          src="https://www.flaticon.com/svg/static/icons/svg/3618/3618186.svg"
          style="height: 70px;"
        />
      </a>
      <br />
       
      <a
        href="/Kategori_/Kitap/En-Yeniler/10001/3"
      >
        <span>
          Yeni Çıkan
          <br />
           Kitaplar
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 8%;"
    >
      <a
        href="/yazarlar"
      >
        <img
          class="icon-kitap"
          src="https://www.dr.com.tr/Themes/DR/Content/assets/images/general/yazarlar-icon.jpg"
          style="height: 80px;"
        />
      </a>
      <br />
       
      <a
        href="/yazarlar"
      >
        <span>
          Yazarlar 
          <span
            style="visibility: hidden;"
          >
            xxxx
          </span>
        </span>
      </a>
    </li>
     
    <span
      class="vl"
      style="font-size: 1000%;"
    />
     
    <li
      style="width: 8%;"
    >
      <a
        href="/yayinevleri"
      >
        <img
          class="icon-kitap"
          src="https://www.dr.com.tr/Themes/DR/Content/assets/images/general/yayinevleri-icon.jpg"
          style="height: 80px;"
        />
      </a>
      <br />
       
      <a
        href="/yayinevleri"
      >
        <span>
          Yayınevleri 
          <span
            style="visibility: hidden;"
          >
            xxxx
          </span>
        </span>
      </a>
    </li>
  </ul>
</div>
`;
